js/tiles.js

```diff
@@ -342,4 +342,19 @@ const door = {
       this.ground.show();
     },
   }),
+      new: Object.assign({}, OpenDoors, {
+  createImages: function() {
+    this.wallTop = this.createImage("tiles/rooms/door/top.svg");
+    this.wallRight = this.createImage("tiles/rooms/door/right.svg");
+    this.ground = this.createImage("tiles/rooms/floor/new.svg");
+  },
+}),
+plain: Object.assign({}, OpenDoors, {
+createImages: function() {
+  this.wallTop = this.createImage("tiles/rooms/door/top.svg");
+  this.wallRight = this.createImage("tiles/rooms/door/right.svg");
+  this.ground = this.createImage("tiles/rooms/floor/plain.svg");
+},
+}),
+};
 };
```

Comment:
fix the errors

Reply:
![image](https://user-images.githubusercontent.com/14032427/34519524-755e0b10-f0aa-11e7-9a92-fac9a0632a8d.png)